InTune/InTune/ViewControllers/ExploreViewController.swift

```diff
@@ -15,48 +15,31 @@ class ExploreViewController: UIViewController {
     @IBOutlet private var artistTableView: UITableView!
     @IBOutlet private var featuredArtistCV: UICollectionView!
     
+    let tabsCVDelegate = TagsCVDelegate()
+    let featuredCVDelegate = FeaturedArtistCVDelegate()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        tagsCollectionView.delegate = self
-        tagsCollectionView.dataSource = self
+        tagsCollectionView.delegate = tabsCVDelegate
+        tagsCollectionView.dataSource = tabsCVDelegate
         artistTableView.delegate = self
         artistTableView.dataSource = self
         artistTableView.register(ExploreArtistCell.self, forCellReuseIdentifier: "exploreCell")
-//        featuredArtistCV.register(UINib(nibName: "FeaturedArtist", bundle: nil), forCellWithReuseIdentifier: "featuredArtist")
-//        featuredArtistCV.delegate = self
-//        featuredArtistCV.dataSource = self
+        featuredArtistCV.register(UINib(nibName: "FeaturedArtist", bundle: nil), forCellWithReuseIdentifier: "featuredArtist")
```

Comment:
let's create a function called 

```
func configurefeaturedArtistCV() {
    featuredArtistCV.register(UINib(nibName: "FeaturedArtist", bundle: nil), forCellWithReuseIdentifier: "featuredArtist")
    featuredArtistCV.delegate = featuredCVDelegate
    featuredArtistCV.dataSource = featuredCVDelegate
}
```

similarly one for `tagsCollectionView `. We should always create a function that will enclose the logic for certain things if there are multiple things happening in a single function.